Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -11845,6 +11912,32 @@ def comma_separated_mapping_to_dict(raw_text):
     return mapping_dict
 
 
+class ManagedSleep:
```

Comment:
No, the intention is to create it once when the integration begins to run, and then for each sleep it will take into account how much time passed since the script started running.